src/multiple_choice/config.py

```diff
@@ -0,0 +1,81 @@
+# Multiple Choice for Anki
+#
+# Copyright (C) 2018-2021  zjosua <https://github.com/zjosua>
+#
+# This file is based on config.py from Glutanimate's
+# Image Occlusion Enhanced Add-on for Anki
+#
+# Copyright (C) 2016-2020  Aristotelis P. <https://glutanimate.com/>
+# Copyright (C) 2012-2015  Tiago Barroso <tmbb@campus.ul.pt>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version, with the additions
+# listed at the end of the license file that accompanied this program.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+# NOTE: This program is subject to certain additional terms pursuant to
+# Section 7 of the GNU Affero General Public License.  You should have
+# received a copy of these additional terms immediately following the
+# terms and conditions of the GNU Affero General Public License that
+# accompanied this program.
+#
+# If not, please request a copy through one of the means of contact
+# listed here: <https://glutanimate.com/contact/>.
+#
+# Any modifications to this file must keep this entire header intact.
+
+"""
+Sets up configuration
+"""
+
+# Compare semantic version: https://stackoverflow.com/a/11887885
+from distutils.version import LooseVersion
```

Comment:
@zjosua Why did you decide against `packaging.version.parse` (from your [stackoverflow answer](https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python/11887885#11887885))?

Both packages don't seem to be included with Anki so we need a dependency folder in the repository structure. I'd recommend to use `packaging`.

Reply:
I forgot that most people don0t have python installed.
Yes, since we need to include a dependency anyway, it's better to use  `packaging`.

Reply:
Ok, done. Any other oversights?